enterprise-suite/templates/es-console-configmap.yaml

```diff
@@ -18,12 +18,12 @@ data:
 
       # use external resolver to lookup backends, cache for 30 seconds
 
-      resolver 127.0.0.1:5353 ipv6=off valid=30s;
+      resolver $KUBE_RESOLVER ipv6=off valid=30s;
 
-      set $prometheus "prometheus-server";
-      set $monitorapi "es-monitor-api";
-      set $grafana "grafana-server";
-      set $alertmanager {{ splitList "," .Values.alertManagers | first | quote }};
+      set $prometheus "prometheus-server.$KUBE_DOMAIN";
+      set $monitorapi "es-monitor-api.$KUBE_DOMAIN";
+      set $grafana "grafana-server.$KUBE_DOMAIN";
+      set $alertmanager {{ print (splitList "," .Values.alertManagers | first) `.$KUBE_DOMAIN:9093` | quote }};
```

Comment:
Any way to parameterize `9093`?

Reply:
I mean what if the list is user-supplied and the first entry happens to run on a different port?  Do we maybe have to somehow do something like:
```
if value contains a port
   use port
else
   use default of 9093
```

Reply:
I was avoiding the problem, because the other pieces also don't specify port :). But since we do support external AM, I should fix this. I'll have a think.

Reply:
Okay, I changed this by moving KUBE_DOMAIN into values.yaml. That makes more sense anyways - if a user wants to provide their own alertmanagers, it won't make sense to append KUBE_DOMAIN.